lib/super_digit.rb

```diff
@@ -1,9 +1,14 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
What is `n` when it comes to time complexity.  I think you either mean O(log10 n) or O(n) where n is the number of digits.